test/Datadog.Trace.Tests/Agent/Thrift/ThriftUdpClientTransportTests.cs

```diff
@@ -0,0 +1,155 @@
+#pragma warning disable SA1636 // File header copyright text must match
+// <copyright file="ThriftUdpClientTransportTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+#pragma warning restore SA1636 // File header copyright text must match
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent.Jaeger;
+using Moq;
+using Thrift.Transport;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Agent.Thrift
+{
+    public class ThriftUdpClientTransportTests : IDisposable
```

Comment:
IMO these tests are really not worth anything. They only value is having higher code coverage but they do not guarantee anything. Espcially that `JaegerThriftClientTransport` is almost only an adapter...